README.md

```diff
@@ -47,16 +47,16 @@ If you get this tmpfs mount error, add `--security-opt apparmor:unconfined \` to
 
 ## Changelog
 
-### 2018.02.27
+### 3.9.2-2
 * Use MongoDB 3.4 repo from MongoDB.
 * Update `beta` github branch to `master`.
 * Add `release` branch to github and switch Docker Hub's `latest` tag to `release` branch.
 * Setup tagging in Docker Hub and created tag for 3.9.0.
 * Switch videos volume location from `/usr/lib/unifi-video/data/videos` to `/var/lib/unifi-video/videos`.
 * Use Docker's `ENTRYPOINT` instead of `CMD`.
```

Comment:
I trust you confirmed all of these; I just threw some into my PR as a suggestion.

Reply:
Yeah, this was all from looking at my commits. :)